### DISCUSSION TITLE
Repo Transfer?

### DISCUSSION THREAD

**Discussion by DocDrydenn** [jojo2357/kiwix-zim-updater#23] [Ideas]
I really feel like I'm holding up the "creativity" and forward progress of this script. I'm a 43yo retired US Navy guy with very poor scripting skills/knowledge and very little free time... 

If you're good with it, I think it would be best for me to transfer ownership of this repo to you (@jojo2357) and just become a collaborator. You clearly have the skills and knowledge. Sound good?

**Comment by jojo2357** [comment]
I wont stop you, but it feels wrong. I didnt come here to commandeer the project, I just saw a cool project that I could make even cooler. With that off my chest, I understand why you might feel that way and if you decide to transfer the repo to me I would like to continue to have your input even if it is delayed at times by real life.

BTW You dont have poor scripting skills :upside_down_face:

**Comment by DocDrydenn** [comment]
It's not commandeering if I give it freely.

Then it's settled. I will transfer it to you and stay on as a collaborator. I'll do what I can to help when I can. I still want to use the script since I still maintain a library of ZIMs that need updating.

Side note: Do not hold back any changes/fixes/updates/etc... because you think I might take offense or something. So long as the script continues to update my ZIM library, I will be happy.... well... okay, maybe just toss me a commented out credit somewhere at the top of the code... nothing fancy... a simple one liner like `#Original idea by DocDrydenn`... you know, somewhere below your own credits. LOL

**Comment by DocDrydenn** [comment]
It says you already have a repo with that name.... gonna need you to change the name of your fork before I can send the main one to you. @jojo2357

**Reply by jojo2357** [reply]
Done. I dont want to delete my fork so Im hoping that the rename will be sufficient for minimal chaos

**Comment by DocDrydenn** [comment]
NOGO... it won't transfer because you have a fork from it. Not sure how to proceed.

**Reply by jojo2357** [reply]
I know how, it will take some git magic on my side tho, will lyk in the next few minutes once I get my local copy properly detached

**Reply by DocDrydenn** [reply]
Copy. Standing by...

**Reply by jojo2357** [reply]
I am go for transfer.

**Reply by DocDrydenn** [reply]
Done. You just need to accept.

**Comment by DocDrydenn** [comment]
Cool. Done and done.

**Reply by jojo2357** [reply]
I wonder what will happen to all those update processes, will they freak out after a while? I know gh will leave the redirect for a while...

**Reply by DocDrydenn** [reply]
Dang... didn't even think of that. LOL 

Meh, anyone with that problem will search for it by name and find it... might need to make a note about the transfer on the Readme?
